hphp/runtime/ext/std/ext_std_file.cpp

```diff
@@ -193,7 +193,15 @@ static int accessSyscall(
         return -1;
       }
     }
-    return ::access(File::TranslatePathWithFileCache(path).data(), mode);
```

Comment:
I think FileCache has no meaning, please see the code (File: TranslatePathWithFileCache),the logic is :
`!translated.empty() && access(translated.data(), F_OK) < 0 && StaticContentCache::TheFileCache`

so the physical file does not exist, and then the file cache won't exist.
I don't think this code is meaningful.